src/components/consoles/DeleteItem.jsx

```diff
@@ -0,0 +1,51 @@
+import { useEffect } from 'react'
+import { useDispatch, useSelector } from 'react-redux';
+import { delConsole, deleteConsole, getConsoles } from '../../redux/consoleSlice';
+
+const DeleteItem = () => {
```

Comment:
- [ ] When I try to delete a console that is reserved, this happens:

![image](https://github.com/stephenkati/gaming_console_reservation_front_end/assets/57421823/e5bb19fc-d77f-48a0-9ff8-85ba51ae54cc)

This is because the reservation is getting in the way.

## Solution and extra tips

I solved this by adding `dependent: destroy` to the model association:

![image](https://github.com/stephenkati/gaming_console_reservation_front_end/assets/57421823/f7bc4166-d3bd-4937-8449-6efcb02bdee1)

Result:

![image](https://github.com/stephenkati/gaming_console_reservation_front_end/assets/57421823/ee22082f-88d6-4636-9dc6-eaef89b9c853)

![image](https://github.com/stephenkati/gaming_console_reservation_front_end/assets/57421823/7f46a112-41bd-4faf-9d41-b32bea2e78f5)

Now that you know the fix, kindly implement it so your consoles are able to be deleted even if they have reservations.   👍 

_note:  If case this is a feature and you don't want your consoles to be deleted when they have a reservation, kindly improve the error message (the alert) so your users know what happened._